### PR TITLE
Clean up Wiremock tests

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: advisor/src/test/kotlin/advisors/OssIndexTest.kt [hunk 1]
```diff
@@ -69,6 +68,7 @@ class OssIndexTest : WordSpec({
 
     beforeTest {
         server.resetAll()
+        configureFor(server.port())
```
Comment (mnonnenmacher) [inline advisor/src/test/kotlin/advisors/OssIndexTest.kt:71]: Doesn't this just reduce the risk of race conditions? It sounds like the correct solution would be not to use a global client instance but create local instances instead, I've never worked with WireMock though.

Comment (sschuberth) [reply advisor/src/test/kotlin/advisors/OssIndexTest.kt:71]: That's a good question that I cannot answer either, as I've also not worked much with WireMock yet. Maybe @oheger-bosch knows? Anyway, this changed solved [the concrete issue I was seeing with Kotest 5](https://kotlinlang.slack.com/archives/CT0G9SD7Z/p1638023794193000).

Comment (sschuberth) [reply advisor/src/test/kotlin/advisors/OssIndexTest.kt:71]: Anyway, as this is at least better than what we had before, can we proceed with this for now as an immediate fix for the race conditions I currently see?

**Review by mnonnenmacher — DISMISSED** [review]

File: advisor/src/test/kotlin/advisors/OssIndexTest.kt (hunk 1 above)
Comment (sschuberth) [reply advisor/src/test/kotlin/advisors/OssIndexTest.kt]: Thanks to @oheger-bosch I've indeed found a better way that I decided to implement right now.

**Review by mnonnenmacher — APPROVED** [review]

**Pull request merged**